draft-ietf-taps-interface.md

```diff
@@ -252,36 +251,42 @@ principles, themselves an elaboration on the architectural design principles
 defined in {{I-D.ietf-taps-arch}}. The interface defined in this document
 provides:
 
-- Access to a variety of transport protocols, independent of the
-  the Protocol Stacks that will be used at
-  runtime, such that all common features of these protocol
+- Access to a variety of transport protocols, independent
+  of the Protocol Stacks that will be used at
+  runtime. All common features of these protocol
   stacks are made available to the application in a
-  transport-independent way to the degree possible, enabling applications written to a single API
-  to make use of transport protocols in terms of the features they provide;
+  transport-independent way to the degree possible.
+  This enables applications written to a single API
+  to make use of transport protocols in terms of the features 
+  they provide.
 
 - A unified interface to datagram and connection-oriented transports, allowing
-  use of a common API for connection-establishment and closing;
+  use of a common API for connection-establishment and closing.
 
 - Message-orientation, as opposed to stream-orientation, using
   application-assisted framing and deframing where the underlying transport
-  does not provide these;
+  does not provide these.
 
-- Asynchronous Connection establishment, transmission, and reception, allowing
-  concurrent operations during establishment and supporting event-driven
+- Asynchronous Connection establishment, transmission, and reception. 
+  This allows concurrent operations during establishment and event-driven
   application interactions with the transport layer, in line with developments
   in modern platforms and programming languages;
   
-- Selection between alternate network paths that can be informed when additional information is available about the networks over which an endpoint can operate (e.g. Provisioning Domain (PvD) information {{RFC7556}});  
+- Selection between alternate network paths, also allowing applications to be
+  notified additional information is available about the networks over which 
```

Comment:
English messy: However, I intended ... informing the selection ... rather than the other possibility of informing the app.

Reply:
(will resolve outside this thread, for some reason having a comment here breaks multiline suggestions)